source/bug-report-guidelines.md

```diff
@@ -56,7 +56,7 @@ The goal of reproducible steps is to teach developer to recreate the bug on his
 
 ### Test Results
 
-The test results, including *Expected Result* and *Actual Result*, will show the developer what's wrong. *Expected Result* describes what should have happened, and *Actual Result* describes what actually happened.
+The test results, including *Expected Result* and *Actual Result*, will tell developers what's wrong. *Expected Result* describes what should have happened, and *Actual Result* describes what actually happened.
```

Comment:
what should happen, & what actually happens.
I think we are describing the common fact, it is better to use the present tense.